tsb/fqdn/gcp/main.tf

```diff
@@ -11,5 +11,5 @@ module "register_fqdn" {
   fqdn          = var.fqdn
   address       = var.address
   project_id    = local.infra["outputs"].project_id
-  vpc_id        = local.infra["outputs"].vpc_id
```

Comment:
also can you please deprecate ```local.infra``` for tsb/fqdn/gcp as well, agree on standardizing using ```data.terraform_remote_state``` across the environments.

Reply:
Done